src/ol/interaction/drawinteraction.js

```diff
@@ -715,6 +715,29 @@ ol.interaction.Draw.prototype.abortDrawing_ = function() {
 
 
 /**
+ * Continue drawing a feature.  This only works on line features, that is
+ * features with a `LINE_STRING` geometry.
+ * @param {!ol.Feature} feature Feature to continue drawing.
```

Comment:
I don't want the `extend` function to assume that `null`may be passed by the user. So by using `!ol.Feature` the user is responsible for providing a feature that is not `null`.